linode_api4/groups/monitor.py

```diff
@@ -145,3 +148,139 @@ def create_token(
                 "Unexpected response when creating token!", json=result
             )
         return MonitorServiceToken(token=result["token"])
+
+    def alert_definitions(
+        self,
+        *filters,
+        service_type: Optional[str] = None,
+    ) -> Union[PaginatedList]:
```

Comment:
```suggestion
    ) -> PaginatedList:
```
You wouldn't need `Union` if there is only one type...